### ISSUE TITLE
hello world?

### ISSUE THREAD

**Issue by gabrielsroka** [okta/okta-sdk-python#123]
can we update the README with a hello world? for example, i'm trying to get this to work, but it's not working

```python
from okta.client import Client as OktaClient
import asyncio

async def main():
    client = OktaClient()
    users, resp, err = await client.list_users()

loop = asyncio.get_event_loop()
loop.run_until_complete(main())
```

**Comment by bretterer** [comment]
@gabrielsroka I will add this to our list to take a look at, however, until then, here is a working example for you to take a look at.

```py
import asyncio
from okta.client import Client as OktaClient
import okta.models as models

async def main():
    my_okta_client = OktaClient()

    usr_list, resp, err = await my_okta_client.list_users()
    print(err)
    print(f"How many users in this list? {len(usr_list)}")

asyncio.run(main())
```

**Comment by bretterer** [comment]
This example shows you not passing anything into your `OktaClient`, so it relies on your `~/.okta/okta.yaml` file for information

**Comment by gabrielsroka** [comment]
i'm also getting
```
ModuleNotFoundError: No module named 'okta.client'
```

i tried it in 2 separate machines. i tried uninstall the old, beta okta package, too

**Comment by bretterer** [comment]
The error you were experiencing in this issue should be resolved with an install of okta through pip with version 1.0.1

**Comment by gabrielsroka** [comment]
with version 1.0.1, this seems to work

```python
from okta.client import Client as OktaClient
import asyncio

async def main():
    client = OktaClient()
    users, resp, err = await client.list_users()
    print(len(users))

loop = asyncio.get_event_loop()
loop.run_until_complete(main())
# asyncio.run(main()) # gives an error
```

**Comment by gabrielsroka** [comment]
here's a hello world for pagination. the example in the readme is confusing. it seems to be 3 different examples (which is fine, but should be made clearer)

```python
from okta.client import Client as OktaClient
import asyncio

async def main():
    client = OktaClient()
    users, resp, err = await client.list_users()
    while True:
        for user in users:
            print(user.profile.login)
        if resp.has_next():
            users, err = await resp.next()
        else:
            break

loop = asyncio.get_event_loop()
loop.run_until_complete(main())
```

**Comment by bretterer** [comment]
This will be resolved by #128
